.github/actions/owners.ts

```diff
@@ -102,4 +102,5 @@ export const REVIEWERS: Record<string, string[]> = {
   turnstile: ["migueldemoura", "punkeel", "worenga"],
   vectorize: ["elithrar", "pdwittig"],
   "waiting-room": ["aolache"],
+  "workers-ai": ["irvinebroque", "pdwittig"]
```

Comment:
```suggestion
  "workers-ai": ["irvinebroque", "pdwittig"],
```
Let's keep a comma after the last item to avoid future mistakes.